src/components/button/README.md

```diff
@@ -1,6 +1,13 @@
 # md-button
 
-`md-button` is an HTML `<button>` or `<a>` tag enhanced with styling and animation to match the [spec of a Material Design button](https://www.google.com/design/spec/components/buttons.html).
+`md-button` is an HTML `<button>` or `<a>` tag enhanced with styling and animation to match the
+[spec of a Material Design button](https://www.google.com/design/spec/components/buttons.html).
```

Comment:
Would it sound better to change `spec of a Material Design button` to `Material Design button spec`?  Current wording sounds awkward.